### DISCUSSION TITLE
Jamulus only works with VPN enabled

### DISCUSSION THREAD

**Discussion by SoWhat0013** [jamulussoftware/jamulus#1944] [Technical Support & Help] [answered]
My Jamulus only works when I enable my NordVpn. I use IOS on my MacBook.

Any suggestion how to solve this so I'm able to use Jamulus without my VPN?

I'm new in this community, so when I submitted this question in the wrong environment ..... sorry for this!

Regards Ton

**Comment by ghost** [comment]
My generalized, non-operating-system-specific suggesion is:
- Stop and start Jamulus after the VPN state has been changed.
- Get to know the VPN very well because Jamulus is not at fault.

**Comment by ann0see** [comment, marked as answer]
Hi Ton,

I assume you’re using macOS, not iOS on your Mac (as far as I know, you can’t install iOS on a Mac yet). 

I’d start debugging as follows:
1. Check if a "kill switch" is enabled in your VPN. Sometimes this kind of software blocks internet access if VPN is disabled
2.  If that‘s not the case check, if you enabled a firewall on your Mac. What about anti virus software: 

Check if you can open http://portquiz.net:22124/ and http://portquiz.net with safari.

If you can’t connect to the 22124 version, but the one without the custom port number, something in your network is blocking port 22124 which Jamulus uses. 
3. Check your router for blocked ports

**Reply by SoWhat0013** [reply]
Thanks for your response. I tried your suggestions without a result.

At the end I called my internet provider and there was a problem with a security setting with respect to port 22124. 

It has been solved now!
